05-JWT-Basics/starter/middleware/auth.js

```diff
@@ -0,0 +1,22 @@
+const jwt = require('jsonwebtoken')
+const {UnauthenticatedError} = require('../errors')
+
+const authenticatedMiddleware = async (req, res, next) => {
+    const authHeader = req.headers.authorization;
+    if (!authHeader || !authHeader.startsWith('Bearer')){
+        throw new UnauthenticatedError('No token provided')
+    }
+
+    const token = authHeader.split(' ')[1]
+
+    try {
+        const decoded = jwt.verify(token, process.envJWT_SECRET)
```

Comment:
```suggestion
        const decoded = jwt.verify(token, process.env.JWT_SECRET)
```

What happens when we call `jwt.verify`? Does a network request happen? How do we know that the JWT is valid?

I also see a typo - `envJWT_SECRET` -> `env.JWT_SECRET`